app/models/vocab_sheet.rb

```diff
@@ -2,11 +2,93 @@
 
 ##
 # A "sheet" of items (signs) saved by a user
+#
 class VocabSheet < ApplicationRecord
-  has_many :items, dependent: :destroy
+  ##
+  # @param item [Item] The item you wish to add
+  # @return [Boolean] true on success, false on failure
+  #
```

Comment:
Are you a method documentation convert @eoinkelly ?

Reply:
Kinda :-) I was trying to mimic the existing API that the controllers depended on so I needed to get the types straight in my head.